documentation/source/newsfragments/README.rst

```diff
@@ -0,0 +1,39 @@
+:orphan:
+
+Release Notes
+=============
+
+We are using `towncrier <https://pypi.org/project/towncrier/>`_ to handle
+our release notes, and this directory contains the input for it -
+"news fragments" which are short files that contain a small
+**ReST**-formatted text that will be added to the next version's
+Release Notes page.
+
+Each file should be named like ``<ISSUE_OR_PR>.<TYPE>.rst``,
+where ``<ISSUE_OR_PR>`` is an issue or a pull request number -
+whatever is most useful to link to,
+and ``<TYPE>`` is one of:
+
+* ``feature``: New user-facing feature.
+* ``bugfix``: A bug fix.
+* ``doc``: Documentation improvement.
+* ``removal``: Deprecation or removal of public API or behavior.
+* ``change``: A change in public API or behavior.
+
+In that file, make sure to use full sentences with correct case and punctuation,
+and do not use a bullet point at the beginning of the file.
+Use Sphinx references (see https://sphinx-tutorial.readthedocs.io/cheatsheet/)
+if you refer to added classes, methods etc.
+
+An example file could consist of the content between the marks:
+
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+Summary of a new feature.
+
+This is a second paragraph.
+<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
+
+Note that the last paragraph should be a normal sentence and not e.g. code,
+because the issue number is appended there.
+
+Towncrier runs automatically when you run any of the Sphinx output build commands.
```

Comment:
I thought I commented on this but maybe I forgot to submit the comment.

```suggestion
Towncrier automatically assembles a list of unreleased changes when building sphinx.

When performing a release, it should be run independently, which will delete all the merged newsfragments, and create new commits.
```
or something.

Reply:
Added, thanks.